### PR TITLE
BET-3760: Added datasource for appliances.

### PR DESCRIPTION
**Pull request by nikchern**

### CODE REVIEW

File: spectrocloud/resource_cluster_libvirt.go [hunk 1]
```diff
@@ -0,0 +1,643 @@
...
+			},
+			"tags": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				Set:      schema.HashString,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+			},
+			"cluster_profile_id": {
```
Comment (rishi-anand) [inline spectrocloud/resource_cluster_libvirt.go:46]: @nikchern We can remove cluster_profile_id and pack

File: spectrocloud/resource_cluster_libvirt.go [hunk 2]
```diff
@@ -0,0 +1,643 @@
...
+							Required: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"appliance_id": {
+										Type:     schema.TypeString,
+										Required: true,
+									},
+									"network_type": {
+										Type:     schema.TypeString,
+										Required: true,
```
Comment (rishi-anand) [inline spectrocloud/resource_cluster_libvirt.go:222]: We can add a constraint to allow just `[default, bridge]` in `network_type`

**Review by rishi-anand — APPROVED** [review]

**Review by rishi-anand — APPROVED** [review]

File: spectrocloud/resource_cluster_libvirt.go (hunk 2 above)
Comment (nikchern) [reply spectrocloud/resource_cluster_libvirt.go]: @rishi-anand added it.

File: spectrocloud/resource_cluster_libvirt.go (hunk 1 above)
Comment (nikchern) [reply spectrocloud/resource_cluster_libvirt.go]: @rishi-anand I removed it.

**Pull request merged**